python/revng/cli/daemon.py

```diff
@@ -13,37 +14,56 @@ class DaemonCommand(Command):
     def __init__(self):
         super().__init__(("daemon",), "Run revng GraphQL server")
 
-    def register_arguments(self, parser):
-        parser.add_argument("-p", "--port", type=str)
+    def register_arguments(self, parser: ArgumentParser):
+        parser.add_argument("-p", "--port", type=str, default="8000", help="Port to use")
+        parser.add_argument(
+            "--export",
+            action="store_true",
+            help="Don't start the server, only print the variables that would be used to start it",
+        )
```

Comment:
Do we still need this now that we have the production server in-house?

Reply:
I dropped `--export`
However I added an option `--hypercorn-args` to allow some flexibility while running the server (e.g. enable hot reload), which seems the real reason behind the need for `--export`